src/dmd/mtype.d

```diff
@@ -5017,6 +5020,43 @@ extern (C++) final class TypeDelegate : TypeNext
     }
 }
 
+/**
+ * This is a shell containing a TraitsExp that can be
+ * either resolved to a type or to a symbol.
+ *
+ * The point is to allow AliasDeclarationY to use `__traits(getMember)`
+ * directly (VS using a library helper in the past).
+ */
+extern (C++) final class TypeTraits : Type
+{
+    Loc loc;
```

Comment:
This field is not in the C++ header.  It seems unused, is it needed?

Reply:
yes i should use it in the alias sem instead of using the loc from the alias.